From: David Faure <faure@kde.org>
Date: Wed, 01 Jan 2014 18:46:25 +0000
Subject: remove unused include (not found, since not linking to kstyle)
X-Git-Url: http://quickgit.kde.org/?p=kde-workspace.git&a=commitdiff&h=eed6be98535e782b0d7327d57bab5171a55e1538
---
remove unused include (not found, since not linking to kstyle)
---


--- a/kcontrol/krdb/krdb.cpp
+++ b/kcontrol/krdb/krdb.cpp
@@ -45,7 +45,6 @@
 #include <ksavefile.h>
 #include <ktemporaryfile.h>
 #include <KLocalizedString>
-#include <kstyle.h>
 
 #include "krdb.h"
 #ifdef HAVE_X11

